src/components/scale.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import {FormattedDate} from 'react-intl';
+
+import * as AppPropTypes from "../app-prop-types"
+import Value from "./value"
+
+
+const Scale = React.createClass({
+  propTypes: {
+    parameter: AppPropTypes.parameter.isRequired,
+  },
+  render() {
+    const {brackets} = this.props.parameter
+    return (
+      <table className="table table-bordered table-hover table-striped">
+        <tbody>
+          {
+            Object.keys(brackets).sort().reverse().map((startDate, startDateIndex) => {
+              const bracketAtDate = brackets[startDate]
+              return Object.keys(bracketAtDate).map((threshold, thresholdIndex) => {
```

Comment:
Something like 
```js
Object.keys(bracketAtDate).sort((a,b) => Number(a) -  Number(b))
```
 should avoid any double conversion.

The API indeed guarantees that the JSON is in the right order, but (@MattiSG tell me if I'm wrong) we don't have any guarantee that the JS **client** will preserve this order. For instance, `Object.keys(bracketAtDate)` could possibly take the keys in an arbitrary order.

I really don't know if this is just a theoretical risk (as most browsers usually preserve the key order), but I'm pretty sure it's not guaranteed in the spec. We are partially covered by the integration tests, but I'm not sure I want to take this risk.